platform/event/gossiped_event.proto

```diff
@@ -0,0 +1,65 @@
+/**
+ * # Gossiped Event
+ * An event that is sent and received via gossip
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.platform.event;
+
+/*
+ * Hedera Network Services Protobuf
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import "event_payload.proto";
+import "core_event_data.proto";
+
+option java_package = "com.hedera.hapi.platform.event";
+// <<<pbj.java_package = "com.hedera.hapi.platform.event">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * An event that is sent and received via gossip
+ */
+message GossipedEvent {
+  /**
+   * The core event data
+   */
+  CoreEventData core_event_data = 1;
+
+  /**
+   * A node signature.<br/>
+   * The signed hash MUST be created using the same methodology as the hash in an EventDescriptor.<br/>
+   * The signature MUST verify using the public key belonging to the `event_creator`.<br/>
+   * The `event_creator` public key SHALL be read from the address book that corresponds to the event's birth round.<br/>
+   * <blockquote>Future work<br/><blockquote>
+   * The precise hash methodology hasn't been determined yet. This documentation must be updated once it has.</blockquote></blockquote>
+   * <blockquote>Future work<br/><blockquote>
+   * The type of signature to use hasn't been determined yet. This documentation must be updated once it has.</blockquote></blockquote>
```

Comment:
(very minor) It's possible to simplify when there are multiple items as follows
```suggestion
   * <blockquote>Future work<br/><blockquote>
   * The precise hash methodology hasn't been determined yet. This documentation must be updated once it has.</blockquote></blockquote>
   * <p>
   * The type of signature to use hasn't been determined yet. This documentation must be updated once it has.</blockquote></blockquote>
```

Reply:
Noted that the second `Future work` line is redundant, but is your suggestion correct in having 4 `</blockquote>` close tags, but only 2 `<blockquote>` open tags?

Reply:
Oops, missed that.  Should only be one pair of closing tags.